app/workers/project_classifications_count_worker.rb

```diff
@@ -0,0 +1,25 @@
+class ProjectClassificationsCountWorker
+  include Sidekiq::Worker
+
+  sidekiq_options congestion: {
+    interval: 60,
```

Comment:
Do we want to make these configurable via env variables with defaults? E.g. [this](https://github.com/zooniverse/Panoptes/blob/233d839b0c1f3b4d2759beede4f4aab658e0f159/config/initializers/panoptes.rb#L9)